templates/csharp/declarative-agent-with-action-from-scratch-bearer/.{{NewProjectTypeName}}/README.md

```diff
@@ -27,9 +27,9 @@
 1. If you haven't added your own API Key, please follow the above steps to add your own API Key.
 2. In the debug dropdown menu, select Dev Tunnels > Create a Tunnel (set authentication type to Public) or select an existing public dev tunnel
 </br>![image](https://raw.githubusercontent.com/OfficeDev/TeamsFx/dev/docs/images/visualstudio/debug/create-devtunnel-button.png)
-3. Right-click your project and select Teams Toolkit > Prepare Teams App Dependencies
-4. If prompted, sign in with a Microsoft 365 account for the Teams organization you want to install the app to.
-5. Press F5, or select the `Debug > Start Debugging` menu in Visual Studio to start your app
+1. Right-click the 'TeamsApp' project in Solution Explorer and select **Teams Toolkit > Select Microsoft 365 Account**
```

Comment:
These numbers don't seem quite right.

Reply:
Yes, I didn't update all the numbers since there're too many files and the steps could change frequently. But it doesn't affect the markdown preview. I can update if you think it's necessary.
![image](https://github.com/user-attachments/assets/7470193e-e425-41e0-8274-9f89a8f318e2)

Reply:
I think it would be better to update this because the preview panel doesn't open automatically in VS, and our users will notice this issue.

Reply:
Updated